### ISSUE TITLE
Problem with short sequences - words

### ISSUE THREAD

**Issue by aleksandarmijat** [daniel-kukiela/nmt-chatbot#113]
Hello. 

I am using this version of NMT to map certain question to it's answer. It works great with sentences. For example, it can give the right answer to a question that's not the same as I imported it (typos, words rearranged etc.). But I want to map word to word. Like I want to map word 'monday' (question) to word 'monday' (answer) so I could detect typos in words. If you input something like 'mnday' i want output to be 'monday' but that doesn't happen, output is some random word.

Is there a way I can modify hparams or something else so this network could work better with short sentences (words)?

**Comment by kaljitism** [comment]
I don't think that this NMT approach will be nice for your task. You need to train a custom model which classifies the input tokens into contains_typo or no_typo, and then if no_typo: then maps the correct most similar vector in terms of Alphabet. 
Research on the Google's AutoCorrrect models and papers.
